plugins/cloudtrail/README.md

```diff
@@ -117,7 +117,7 @@ We describe each of these below.
 
 When using `s3://<S3 Bucket Name>/[<Optional Prefix>]`, the plugin will scan the bucket a single time for all objects. Characters up to the first slash/end of string will be used as the S3 bucket name, and any remaining characters will be treated as a key prefix. After reading all objects, the plugin will return EOF.
 
-All objects below the bucket, or below the bucket + prefix, will be considered cloudtrail logs. Any object ending in .json.gz will be decompressed first.
+All objects below the bucket, or below the bucket + prefix, will be considered cloudtrail logs. Any object ending in .json.gz will be decompressed first. The plugin expects the logs to be compressed.
```

Comment:
The test we made was to put plain text `.json` files so the plugin could read and the files were ignored by the plugin. Once we put them in `.gz` files, the plugin worked seamlessly.

Reply:
By looking at the code, the plugin seems to handle also plain text `.json` files. Maybe a bug?

Let's ask @ldegio :smile_cat: 
What's the expected behavior here?

Reply:
In order to give you guys more evidence:
- Log containing **DescribeInstances** `eventName`:
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/50663919/166002553-9145e7d6-696f-466e-8d6e-8affd8d39b6c.png">

- Mock Falco rule to trigger it:
```YAML
- rule: Lucas Teixeira Rule II
  desc: Lucas Teixeira has been disabled, this could be potentially malicious.
  condition:
    ct.name="DescribeInstances"
  output:
    Lucas Teixeira II just triggered!
  priority: WARNING
  tags:
    - cloud
    - aws
    - aws_cloudtrail
  source: aws_cloudtrail
```

- Log file on the bucket in JSON:
![image](https://user-images.githubusercontent.com/50663919/166003249-b570b711-26f4-4cb2-83cd-5131200ef066.png)

- Falco:
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/50663919/166003359-d78b11cd-c203-4a7b-842c-328e99128bc6.png">

- Now, log file in the bucket compressed in `.gz`:
![image](https://user-images.githubusercontent.com/50663919/166003657-d998595c-e46f-4fb3-b2ff-b9065d0eb95f.png)

- Falco:
<img width="1176" alt="image" src="https://user-images.githubusercontent.com/50663919/166003787-19b5f683-5bb4-4ce1-bcdd-9592e9be1f1b.png">

Reply:
plain text json files are expected to be supported. I just ran a quick test on my machine and the plugin was able to pick up and decode a directory with 2 plain text files.

Reply:
Giving you guys more evidence of the issue I've faced. 
- I have come up with this mock rule to make sure something will trigger:
```YAML
- rule: Lucas Teixeira Rule II
  desc: Lucas Teixeira has been disabled, this could be potentially malicious.
  condition:
    ct.name="DescribeInstances"
  output:
    Lucas Teixeira II just triggered!
  priority: WARNING
  tags:
    - cloud
    - aws
    - aws_cloudtrail
  source: aws_cloudtrail
```

- Mock log file in plain `json` containing multiple `DescribeInstances` events in it:
![image](https://user-images.githubusercontent.com/50663919/166005339-02fd76ea-c1a0-4b91-b1d1-294380663a61.png)

- Falco:
<img width="1160" alt="image" src="https://user-images.githubusercontent.com/50663919/166005486-0cb47ae5-09d4-46df-bef8-066237bae35c.png">

- Same log file, now compressed in `.gz`:
![image](https://user-images.githubusercontent.com/50663919/166005734-32af5327-3927-4bac-bba4-5a9af17249ed.png)

- Falco:
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/50663919/166006253-0f9ab655-e49d-446b-8a56-20b8f37a1106.png">

Reply:
Hey @exiett 

Thank you for having reported your tests. We need to investigate a bit, I believe there could be a bug.
I'll keep you posted.

Reply:
I was able to reproduce this behavior. I can confirm that with the latest version of the cloudtrail plugin:
* plain text files supplied locally (e.g. `/tmp/my_logs.json`) work fine
* gzipped text files supplied in a bucket (e.g. `s3://my-bucket/logs/`) work fine
* plain text files supplied in a bucket do not generate detections